sample-data/Samples/Sample XML/BearCreek-StudentEnrollment.xml

```diff
@@ -2689,7 +2796,7 @@
 		<EntryDate>2021-08-23</EntryDate>
 		<EntryGradeLevel>uri://ed-fi.org/GradeLevelDescriptor#First grade</EntryGradeLevel>
 		<GraduationPlanReference ref="GPLN_Minimum-BCES"/>
-		<FullTimeEquivalency>0.75</FullTimeEquivalency>
```

Comment:
@vimayya Example of one changing from 0.75 to 1.00.

Reply:
@blmeyers @markramonDL If there was a value before, let us not change it.

Reply:
restored original values.